lemur/certificates/views.py

```diff
@@ -1155,6 +1155,7 @@ def get(self, notification_id):
         )
         parser.add_argument("creator", type=str, location="args")
         parser.add_argument("show", type=str, location="args")
+        parser.add_argument("showExpired", type=int, location="args")
```

Comment:
good catch @jtschladen.
would it make sense to default this to only not expired certs?
From user experience, probably only notifications on not expired certs would add value, as while editing the notification, you can drop them.

from Performance point of view, it would make a huge difference searching the entire DB or only valid certs.

Reply:
Added this, although I'd like to note for the record that hiding expired certs means those expired certs will not longer be associated with notifications after editing. I think this is fine, as we won't notify on expired certs anyway.